### PR TITLE
Feature checkout information

### PR DESCRIPTION
**Pull request by condef5**
### Status
Required

### Implementation Notes
* Refactor some styles
* Added navbar fixed.
* Listed details information in checkout

### CODE REVIEW

File: client/src/views/checkout.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    getSportField(cart.SportField).then(data => {
+      console.log(data);
+      setSportField(data);
+      setLoading(false);
+    });
+  }, []);
+
+  function format(hour) {
+    return ("0" + hour.toString() + ":00").slice(-5);
+  }
```
Comment (cristianbgp) [inline client/src/views/checkout.js]: That's a cool way to display the formated hours

File: client/src/components/ui.js [hunk 2]
```diff
@@ -313,6 +313,27 @@ function TextArea({ styles, ...props }) {
   return <textarea {...props} css={{ ...styleInput, ...styles }} />;
 }
 
+const styleScroll = {
```
Comment (cristianbgp) [inline client/src/components/ui.js]: Thanks for moving this style for the scroll to the `ui.js` file

File: client/src/components/spinner.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+import { jsx, css, keyframes } from "@emotion/core";
+
+function Spinner() {
+  const bounce = keyframes`
+    0%, 80%, 100% { 
+      transform: scale(0);
+    } 40% { 
+      transform: scale(1.0);
+    }
+  `;
```
Comment (cristianbgp) [inline client/src/components/spinner.js]: I didn't know thats the way to create animation keyframes :open_mouth:

File: client/src/components/ui.js (hunk 2 above)
Comment (condef5) [reply client/src/components/ui.js]: You're welcome `:)`

**Pull request merged**